### PR TITLE
add ADATW 

### PR DESCRIPTION
**Pull request by linuxjason**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -2430,5 +2430,20 @@
...
+    "social": {
+      "twitter": "",
+      "facebook": "",
+      "reddit": "",
+      "instagram": "",
+      "website": "",
+      "youtube": "https://www.youtube.com/channel/UCu--wp47a20YMJbrzJmtv2w"
+    },
+    "telegram": "Tw_JasonLai",
+    "operatorName": "linuxjason",
```
Comment (leo42) [inline registry.json:2446]: We need a real name or leave it empty please.

**Review by leo42 — CHANGES_REQUESTED** [review]

**Comment by Bez625** [comment]
@linuxjason Please can you address the review comments?

File: registry.json (hunk 1 above)
Comment (linuxjason) [reply registry.json:2446]: Please change to Jason thanks

Comment (Bez625) [reply registry.json:2446]: @linuxjason please can you change in the PR? Thanks

**Pull request merged**